src/icon/colorIcon.tsx

```diff
@@ -0,0 +1,25 @@
+import React from 'react'
+
+import { BaseIcon, BaseIconDefaultProps, Icon } from '../_utils/icon'
+
+export type ColorIconProps = Icon &
+  Readonly<{
+    borderColor?: string
+  }>
+
+export const ColorIcon = (props: ColorIconProps) => (
+  <BaseIcon {...props}>
+    <circle
+      cx="12"
+      cy="12"
+      r="9.5"
+      stroke={props.borderColor || props.iconColor}
```

Comment:
please use **CSS**(in-js) for that

Reply:
Same answer as my previous PR.

Reply:
not the same issue! here is even more problematic since we are treating fill and stroke